spec/spec_helper_acceptance.rb

```diff
@@ -29,6 +29,9 @@
         on host, 'sed -i "s/keepcache=.*/keepcache=1/" /etc/yum.conf'
         # refresh check if cache needs refresh on next yum command
         on host, 'yum clean expire-cache'
+
+        host.install_package('centos-release-scl')
```

Comment:
Small optimization: it's sufficient to only install the RH set of SCLs and that's also what we do in real deployments.
```suggestion
        host.install_package('centos-release-scl-rh')
```

Reply:
Thanks, makes sense :)